core/server/models/post.js

```diff
@@ -674,6 +674,13 @@ Post = ghostBookshelf.Model.extend({
             }
         }
 
+        // newsletter_id is read-only and should only be set using a query param when publishing/scheduling
```

Comment:
What is it that makes the newsletter_id read only? afaict you could still override it if you include it in the body

Reply:
At the moment the `newsletter_id` is neither in the input nor in the output of the post API, so including it in the body will return an error.
Could you point me how we're doing it for https://github.com/TryGhost/Ghost/blob/d7a450559f843175ca78d8818eda8202d1c8521e/core/server/models/post.js#L684 ? I think it would be good that I implement a similar approach in case we choose to add `newsletter_id` to the post api.

Reply:
It's stripped in the API schema [here](https://github.com/TryGhost/SDK/blob/1f90c51a5b34e86fae5c1b6c3b56497f30462f63/packages/admin-api-schema/lib/canary/posts.json#L188-L190)